File: test_clients/C#/day_trader_testclient/DayTraderTestClient/DayTraderTestClient.Common/DTO/Positions/PositionNewReq.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DayTraderTestClient.Common.DTO.Positions
{
    public class PositionNewReq : WebSocketMessage
    {
        #region Public Attributes

        public string UUID { get; set; }

        public string ReqId { get; set; }

        public string Symbol { get; set; }

        public string SecurityType { get; set; }

        public string Exchange { get; set; }


        #endregion
    }
}

File: day_trader/sources/strategy/strategies/day_trader/strategy/services/websocket/common/DTO/positions/position_new_req.py
from sources.strategy.strategies.day_trader.business_entities.day_trading_position import *
from sources.strategy.strategies.day_trader.strategy.services.websocket.common.DTO.websocket_message import *
import json

class PositionNewReq(WebSocketMessage):


    def __init__(self, Msg,ReqId,Symbol=None,SecurityType=None,Exchange=None,UUID=None):
        super(PositionNewReq, self).__init__(Msg)

        self.Symbol=Symbol
        self.UUID = UUID
        self.ReqId = ReqId
        self.SecurityType=SecurityType
        self.Exchange=Exchange

    def toJSON(self):
        return json.dumps(self, default=lambda o: o.__dict__,sort_keys=True)